### PR TITLE
Replace wallet with polkadot/keyring

### LINKED ISSUE #170: Remove wallet
We have decided not to support CENNZnet wallet functionality. Our API users would have no issues using Polkadot keyring for the same use cases.

### PR DESCRIPTION
**Pull request by alexsednz**
Closes #170

### CODE REVIEW

File: docs/examples/rx/06_make_transfer/package.json [hunk 1]
```diff
@@ -10,15 +10,7 @@
     "start": "node index.js"
   },
   "dependencies": {
-    "@cennznet/api": "^1.0.3",
-    "@cennznet/wallet": "^1.0.3"
-  },
-  "resolutions": {
```
Comment (holygits) [inline docs/examples/rx/06_make_transfer/package.json:16]: I think the resolutions are still relevant?

File: packages/api/test/e2e/queries.e2e.ts [hunk 2]
```diff
@@ -22,11 +22,13 @@ import { cryptoWaitReady } from '@polkadot/util-crypto';
...
 describe('e2e queries', () => {
-  let api, alice, bob;
+  let api, alice, aliceControllerCennznet, aliceStashCennznet, bob;
 
   beforeAll(async () => {
     await cryptoWaitReady();
-    const keyring = new Keyring({ type: 'sr25519' });
+    const keyring = testKeyring();
+    aliceControllerCennznet = "5GrwvaEF5zXb26Fz9rcQpDWS57CtERHpNehXCPcNoHGKutQY";
+    aliceStashCennznet = "5GNJqTPyNqANBkUVMN1LPPrxXnFouWXoe2wNSmmEoLctxiZY";
```
Comment (holygits) [inline packages/api/test/e2e/queries.e2e.ts:31]:
```suggestion
    const keyring = testKeyring({ type: 'sr25519' });
    let aliceController = keyring.addFromUri('//Alice');
    let aliceStash = keyring.addFromUri('//Alice//stash')
```

**Review by holygits — COMMENTED** [review]
thanks for this!

File: docs/examples/rx/06_make_transfer/package.json (hunk 1 above)
Comment (alexsednz) [reply docs/examples/rx/06_make_transfer/package.json:16]: Yup right, it slipped in from the original cherrypick. I have restored it. Thank you.

**Review by KarishmaBothara — APPROVED** [review]
Nice work..

**Pull request merged**